lib/internal/crypto/keys.js

```diff
@@ -299,8 +299,7 @@ function isStringOrBuffer(val) {
 }
 
 function parseKeyEncoding(enc, keyType, isPublic, objName) {
-  if (enc === null || typeof enc !== 'object')
-    throw new ERR_INVALID_ARG_TYPE('options', 'object', enc);
+  validateObject(env, 'options');
```

Comment:
enc

Reply:
yep, autocorrect screwed that right up 😅